setup/providers/protonvpn.md

```diff
@@ -72,9 +72,12 @@ services:
 
 Requirements:
 
-- Add `+pmp` to your OpenVPN username (thanks to [@mortimr](https://github.com/qdm12/gluetun/issues/1760#issuecomment-1669518288))
 - `VPN_PORT_FORWARDING=on`
-- If you use **Wireguard** using the custom provider, set `VPN_PORT_FORWARDING_PROVIDER=protonvpn`
+- `PORT_FORWARD_ONLY=on` to tell Gluetun to only connect to servers that support port forwarding
+- For OpenVPN
+  - Append `+pmp` to your OpenVPN username. For example, if your ProtonVPN username was `johndoe`, set `OPENVPN_USER` to `johndoe+pmp` (thanks to [@mortimr](https://github.com/qdm12/gluetun/issues/1760#issuecomment-1669518288)). ProtonVPN documentation on [port forwarding with OpenVPN](https://protonvpn.com/support/port-forwarding-manual-setup#openvpn)
```

Comment:
I prefer it shorter, but nice example still 👍 Also nice `Add` replacement with `Append`!

```suggestion
- For OpenVPN only, append `+pmp` to your OpenVPN username (thanks to [@mortimr](https://github.com/qdm12/gluetun/issues/1760#issuecomment-1669518288)). If needed, see the [ProtonVPN OpenVPN port forwarding documentation](https://protonvpn.com/support/port-forwarding-manual-setup#openvpn).
```